crates/base_db/Cargo.toml

```diff
@@ -18,4 +18,3 @@ profile = { path = "../profile", version = "0.0.0" }
 tt = { path = "../tt", version = "0.0.0" }
 test_utils = { path = "../test_utils", version = "0.0.0" }
 vfs = { path = "../vfs", version = "0.0.0" }
-stdx = { path = "../stdx", version = "0.0.0" }
```

Comment:
I'd rather keep `stdx` -- it' uself to `stdx::print_backtrace` when debugging without messing with Cargo.tomls. 

Though, doesn't really matter.